### ISSUE TITLE
release.yml GitHub Action runs on push, then fails

### ISSUE THREAD

**Issue by tchamberlin** [GreenBankObservatory/dysh#69]
I'm not sure why it's getting triggered at all -- I'm pushing to a non-release branch. The `on` block seems sensible at first glance...

```yaml
on:
  release:
    types: [ published ]
  workflow_dispatch:
```

That said, once it _does_ run, it has an error:

```plain
Error: .github#L1
every step must define a `uses` or `run` key
```

Which I _think_ is about the `hatch` step

See e.g. https://github.com/GreenBankObservatory/dysh/actions/runs/5825834159

**Comment by mpound** [comment]
I wouldn't expect (didn't intend) it to run  push at all.   I thought on: release: meant it runs when a new release is created.

**Comment by tchamberlin** [comment]
That's my interpretation of [the docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release), too -- not sure what's going on 🤷

**Comment by tchamberlin** [comment]
Resolved by #72, hopefully
